net/net.go

```diff
@@ -165,11 +165,14 @@ func AddIPOffset(base *big.Int, offset int) net.IP {
 }
 
 // RangeSize returns the size of a range in valid addresses.
-func RangeSize(subnet *net.IPNet) int64 {
+func RangeSize(subnet *net.IPNet, boundary uint) int64 {
 	ones, bits := subnet.Mask.Size()
 	if bits == 32 && (bits-ones) >= 31 || bits == 128 && (bits-ones) >= 127 {
 		return 0
 	}
+	if bits-ones >= int(boundary) {
```

Comment:
rather than passing in a boundary, should we just do `if (bits-ones) >= 63 { return 0 }`?

Reply:
or `if (bits-ones) >= 63 { return math.MaxInt64 }`